### ISSUE TITLE
Packaging; removing tests and fabfile

### ISSUE THREAD

**Issue by prologic** [circuits/circuits#79]
See: #78 

For almost the same reasons do we need:

* [fabfile](https://github.com/circuits/circuits/tree/master/fabfile)
* [tests](https://github.com/circuits/circuits/tree/master/tests)

These get packaged up as part of circuits because of the ``find_packages(".")`` used in [``setup.py``](https://github.com/circuits/circuits/tree/master/setup.py)

**Comment by eriol** [comment]
I would like to propose to not install fabfile and tests, but only ship them in source dist.
But well I'm not sure about shipping fabfile at all, maybe we can exclude them.

Right now installing circuits will pollute namespace:
```shell
❯ ls $HOME/.virtualenvs/circuits/local/lib/python2.7/site-packages/tests
app  conftest.py  conftest.pyc  core  __init__.py  __init__.pyc  io  main.py  main.pyc  net  protocols  tools  web
❯ ls $HOME/.virtualenvs/circuits/local/lib/python2.7/site-packages/fabfile 
docker.py  docker.pyc  docs.py  docs.pyc  help.py  help.pyc  __init__.py  __init__.pyc  utils.py  utils.pyc
```

**Comment by prologic** [comment]
Is this effectively closed by #109 ?

**Comment by eriol** [comment]
Yes, if you want to keep fabfile inside source dist. I see no harm from having them included since we are sure prebuilt documentation will not be shipped by mistake in source dist.
